contracts/curve/types/JumpRateUtilizationCurve.sol

```diff
@@ -48,6 +48,19 @@ library JumpRateUtilizationCurveLib {
         }
         return self.maxRate.unpack();
     }
+
+    function accumulate(
+        JumpRateUtilizationCurve memory self,
+        UFixed18 utilization,
+        uint256 fromTimestamp,
+        uint256 toTimestamp,
+        UFixed18 notional
+    ) internal pure returns (Fixed18) {
+        return compute(self, utilization)
+            .mul(Fixed18Lib.from(int256(toTimestamp - fromTimestamp)))
+            .mul(Fixed18Lib.from(notional))
+            .div(Fixed18Lib.from(365 days));
```

Comment:
instead of `365 days` we can actually just do `1 years`

Reply:
The Solidity compiler tells me `Using "years" as a unit denomination is deprecated`, probably because not every year has 365 days.